src/components/SingleArticle/SingleArticle.stories.tsx

```diff
@@ -15,16 +16,14 @@ export default {
   ],
 } as Meta
 
-const testArticle = {
+const testArticle: ArticleRecord = {
```

Comment:
Because I made status a type of `'Draft' | 'Published' | 'Archived'` in `ArticleRecord` type we have to specify the type wen defining an article with the status. I don't mind too much and typescript compiler lets you know when you need to.